docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,161 @@
+
+![find get activities](../_static/images/howto_guides/api_gui/find_get_activities.png)
+The `GET /activities` endpoint is a general way to fetch metadata about various activities (e.g. metagenome assembly, natural organic matter analysis, library preparation, etc.). Any "slot" (a.k.a. attribute) for [WorkflowExecutionActivty](https://microbiomedata.github.io/nmdc-schema/WorkflowExecutionActivity/) or [PlannedProcess](https://microbiomedata.github.io/nmdc-schema/PlannedProcess/)  may be used in the filter and sort parameters, including attributes for subclasses of `WorkflowExecutionActvity` and `PlannedProcess`, such as slots used in the `MetabolomicsAnalysisActivity` or `Extraction` class among others.<br/>
+<br/>
+
+![find get activities by activity id](../_static/images/howto_guides/api_gui/find_get_activities_activity_id.png)
+If the activity identifier is known, the activity metadata can be retrieved using the `GET /activities/activity_id` endpoint. Note that only one metadata record for an activity may be returned at a time using this method.<br/>
+<br/>
+
+For more information and to see more examples of __find__ endpoints outside of the [autogenerated user interface](https://api.microbiomedata.org/docs#/find), please visit: https://api.microbiomedata.org/search<br/>
+<br/>
+
+#### Find Endpoint Example: get all studies that have EMSL (Environmental Molecular Sciences Laboratory) related funding
+
+1. Click on the drop down arrow to the right side of the **`GET /studies`** endpoint
+![find example step 1](../_static/images/howto_guides/api_gui/find_example_step1.png)
+2. Click **Try it out** in the upper right of the expanded endpoint box
+![find example step 2](../_static/images/howto_guides/api_gui/find_example_step2.png)
+3. Enter in parameters. In this case, we will input `funding_sources.search:EMSL` into the **filter** parameter. The `.search` performs a full text search to find studies with `funding_sources` that have the word "EMSL" in its value.
+4. Click **Execute**
+![find example step 3 and step 4](../_static/images/howto_guides/api_gui/find_example_step3and4.png)
+5. View the results in JSON format, available to download by clicking **Download**, or copy the results by clicking the clipboard icon in the bottom right corner of the response. In this case, two studies were retrieved.
+![find example step 5](../_static/images/howto_guides/api_gui/find_example_step5.png)<br/>
+<br/>
+
+- Note that a curl request and request URL are provided as well for command line usage:
+![find example note](../_static/images/howto_guides/api_gui/find_example_note.png)
+
+#### ___Metadata___ Endpoints
+
+The [metadata endpoints](https://api.microbiomedata.org/docs#/metadata) can be used to get and filter metadata from collection set types (including studies, biosamples, activites, and data objects as discussed in the __find__ section), as well as validate and submit updates to existing metadata to the data portal as a spreadsheet or json file. 
```

Comment:
same question .... and maybe I'm forgetting what our target was. If it's just users ... we don't need the "as well as validate and submit updates to existing metadata to the data portal as a spreadsheet or json file" part.

Reply:
Removed! [c96e9ee](https://github.com/microbiomedata/NMDC_documentation/pull/27/commits/c96e9ee6fa5dc0e3579412152b1e103909a2a897)